src/atomate2/vasp/schemas/calculation.py

```
@@ -828,6 +840,26 @@ def _get_volumetric_data(
     return volumetric_data
 
 
+def _parse_waveder(dir_name: Path) -> Optional[Waveder]:
+    """
+    Parse the WAVEDER file.
+
+    Parameters
+    ----------
+    dir_name
+        The directory containing the WAVEDER file.
+
+    Returns
+    -------
+    Optional[Waveder]
+        The WAVEDER data.
+    """
+    try:
+        return Waveder.from_binary(dir_name / "WAVEDER")
+    except Exception:
```

Could we not catch all exceptions?